components/gitpod-db/src/typeorm/entity/db-identity.ts

```diff
@@ -36,6 +36,15 @@ export class DBIdentity implements Identity {
     @Column()
     deleted?: boolean;
 
+    @Column({
+        default: "",
+        transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED,
+    })
+    lastSigninTime?: string;
```

Comment:
Would be nice to send the list of Identities ordered by the `lastSigninTime` from the backend, but I suspect that could be harder to given the one-to-many relationship

Reply:
that's actually a good idea.